rootfs/etc/nginx/lua/balancer/chashsubset.lua

```diff
@@ -0,0 +1,92 @@
+-- Consistent hashing to a subset of nodes. Instead of returning the same node
+-- always, we return the same subset always.
+
+local resty_chash = require("resty.chash")
+local util = require("util")
+
+local _M = { name = "chashsubset" }
+
+local function build_nodeset_map(backend)
+  local nodes = {}
+  local nodeset_map = {}
+  local node_sets = {}
+  local subset_size = 3
```

Comment:
Can you set this default value in controller side when you parse the annotation? That's how it's done in the rest of the code base, so that we can define default values in one place as much as possible instead of scattering them across Lua and Go files.